# Occitan (post 1500) translation for apport
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-06-06 11:55+0200\n"
"PO-Revision-Date: 2024-09-27 16:45+0000\n"
"Last-Translator: Quentin PAGÈS <Unknown>\n"
"Language-Team: Occitan (post 1500) <oc@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2024-12-06 11:21+0000\n"
"X-Generator: Launchpad (build 39bd251485adde1a3ef538479f6c030babb8e251)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Collectar d'informacions a prepaus del sistèma"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Una autentificacion es necessària per collectar l'informacion del sistèma "
"requesida per aqueste rapòrt de bug"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Rapòrts de problèmas sistèma"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"Picatz vòstre senhal per accedir a de rapòrts de problèmas de programas "
"sistèma"

#: ../apport/ui.py:263
msgid "This package does not seem to be installed correctly"
msgstr "Sembla qu'aqueste paquet es pas installat corrèctament."

#: ../apport/ui.py:273
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""
"Aquò sembla pas un paquet %s oficial. Volgatz tornar ensajar aprèp aver mesa "
"a jorn los indexes dels paquets disponibles, se fonciona pas, suprimissètz "
"los paquets tèrces ligats e tornatz ensajar."

#: ../apport/ui.py:303
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Las versions installadas d'unes paquets son obsoletas. Metètz a jorn los "
"paquets seguents, puèi verificatz se lo problèma es encara present :%s"

#: ../apport/ui.py:454
msgid "unknown program"
msgstr "programa desconegut"

#: ../apport/ui.py:456
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "O planhèm, lo programa « %s » a quitat d'un biais imprevist"

#: ../apport/ui.py:458
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Vòstre ordenador possedís pas pro de memòria liura per analisar "
"automaticament lo problèma e mandar un rapòrt als desvolopaires."

#: ../apport/ui.py:463 ../apport/ui.py:1872
#, python-format
msgid "Problem in %s"
msgstr "Problèma dins %s"

#: ../apport/ui.py:519 ../apport/ui.py:536 ../apport/ui.py:695
#: ../apport/ui.py:701 ../apport/ui.py:931 ../apport/ui.py:1695
#: ../apport/ui.py:1839 ../apport/ui.py:1845
msgid "Invalid problem report"
msgstr "Rapòrt d'incident invalid"

#: ../apport/ui.py:520
msgid "You are not allowed to access this problem report."
msgstr "Sètz pas autorizat a accedir aqueste rapòrt d'incident."

#: ../apport/ui.py:528
msgid "Error"
msgstr "Error"

#: ../apport/ui.py:530
msgid "There is not enough disk space available to process this report."
msgstr "I a pas pro d'espaci de disc disponible per tractar aqueste rapòrt."

#: ../apport/ui.py:569 ../apport/ui.py:674
msgid "Invalid PID"
msgstr "PID invalida"

#: ../apport/ui.py:569
msgid "The specified process ID does not exist."
msgstr "L'ID de processus indicat existís pas."

#: ../apport/ui.py:574
msgid "Not your PID"
msgstr "Pas vòstre PID"

#: ../apport/ui.py:575
msgid "The specified process ID does not belong to you."
msgstr "L’identificant de processus indicat vos aparten pas."

#: ../apport/ui.py:632
msgid "No package specified"
msgstr "Cap de paquet pas especificat"

#: ../apport/ui.py:634
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Vos cal precisar un paquet o un PID. Utilizatz --help per mai d'informacions."

#: ../apport/ui.py:661
msgid "Permission denied"
msgstr "Autorizacion refusada"

#: ../apport/ui.py:663
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Lo processus especificat vos aparten pas. Executatz aqueste programa en tant "
"que proprietari del processus o en tant qu'administrator (root)."

#: ../apport/ui.py:675
msgid "The specified process ID does not belong to a program."
msgstr "L'ID de processus especificat aparten pas a cap de programa."

#: ../apport/ui.py:696
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "L'escript  %s a pas pogut determinar un paquet afectat"

#: ../apport/ui.py:702
#, python-format
msgid "Package %s does not exist"
msgstr "Lo paquet %s existís pas"

#: ../apport/ui.py:731 ../apport/ui.py:936 ../apport/ui.py:969
#: ../apport/ui.py:976
msgid "Cannot create report"
msgstr "Impossible de crear lo rapòrt"

#: ../apport/ui.py:748 ../apport/ui.py:802 ../apport/ui.py:819
msgid "Updating problem report"
msgstr "Mesa a jorn del rapòrt d'incident"

#: ../apport/ui.py:750
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Aqueste rapòrt es pas estat fait per vos o vos i sètz pas inscrit ; o es "
"tampat o s'agís d'un doblon.\n"
"Creatz un rapòrt novèl en utilizant « apport-bug »."

#: ../apport/ui.py:762
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Sètz pas lo redactor d'aqueste rapòrt d'incident. Es pus aisit de marcar un "
"bug coma duplicata que non pas de desplaçar vòstres comentaris e pèças "
"juntas sus un bug novèl.\n"
"Vos recomandam doncas de completar un senhalament de bug novèl en utilizant "
"\"apport-bug\"  e de ne far un comentari dins lo bug inicial.  \n"
"Sètz segur que volètz contunhar ?"

#: ../apport/ui.py:802 ../apport/ui.py:819
msgid "No additional information collected."
msgstr "Cap d'informacion suplementària es pas estada collectada."

#: ../apport/ui.py:875
msgid "What kind of problem do you want to report?"
msgstr "Quin tipe de problèma volètz senhalar ?"

#: ../apport/ui.py:894
msgid "Unknown symptom"
msgstr "Simptòma desconegut"

#: ../apport/ui.py:895
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Lo simptòma « %s » es desconegut."

#: ../apport/ui.py:915
msgid "No PID specified"
msgstr "Cap de PID especificat"

#: ../apport/ui.py:916
msgid "You need to specify a PID. See --help for more information."
msgstr "Devètz especificar un PID. Vejatz --help per mai d’informacions."

#: ../apport/ui.py:938
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""
"L'opcion fenèstra pòt pas èsser utilizada sus Wayland.\n"
"\n"
"Mercés de trobar l'ID de processus de la fenèstra e d'executar alara 'ubuntu-"
"bug <ID de processús>'.\n"
"\n"
"L'identificant del procès se pòt trobar en executant l'aplicacion System "
"Monitor. A l’onglet Processús, desplaçatz-vos entrò trobar l'aplicacion "
"corrècta. L'ID del procès es lo nombre listat dins la colomna ID."

#: ../apport/ui.py:954
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Aprèp la tampadura d'aqueste messatge, clicatz sus la fenèstra de "
"l'aplicacion per la quala volètz senhalar un problèma."

#: ../apport/ui.py:970 ../apport/ui.py:977
msgid "xprop failed to determine process ID of the window"
msgstr "xprop a pas pogut determinar l'ID del processus de la fenèstra"

#: ../apport/ui.py:992
#, python-format
msgid "%(prog)s <report number>"
msgstr "%(prog)s <numèro de rapòrt>"

#: ../apport/ui.py:993
msgid "Specify package name."
msgstr "Indicatz lo nom del paquet."

#: ../apport/ui.py:999 ../apport/ui.py:1127
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr ""
"Apondre un marcador suplementari pel rapòrt. Pòt èsser especificat mantun "
"còp."

#: ../apport/ui.py:1035
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"

#: ../apport/ui.py:1046
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Avia en mòde senhalament de bug. Necessita l'argument --package e --pid en "
"opcion, o simplament --pid. Se n'i a pas cap de provesit, aficha una lista "
"de simptòmas coneguts. (Implicit se un sol argument es provesit.)"

#: ../apport/ui.py:1055
msgid "Click a window as a target for filing a problem report."
msgstr "Clicatz sus la fenèstra per la quala volètz senhalar un problèma."

#: ../apport/ui.py:1063
msgid "Start in bug updating mode. Can take an optional --package."
msgstr "Començar en mòde « mesa a jorn de bug ». Accèpta l'opcion --package"

#: ../apport/ui.py:1071
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Efectua un senhalament de bug a prepaus d'un simptòma. (Implicit se lo nom "
"del simptòma es donat coma argument unic.)"

#: ../apport/ui.py:1080
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Especifica un nom de paquet dins lo mòde --file-bug. Aquò's opcional se --"
"pid es especificat. (Implicit se lo nom del paquet es donat coma argument "
"unic.)"

#: ../apport/ui.py:1091
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Especificar un programa que s'executa en mòde « --file-bug ». S'aquò es "
"especificat, lo rapòrt de bug contendrà mai d'informacions. (implicita se "
"l’identificant del processus es passat coma argument unic)"

#: ../apport/ui.py:1099
msgid "The provided pid is a hanging application."
msgstr "Lo pid provesit es una aplicacion que respond pas."

#: ../apport/ui.py:1107
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Senhala lo bug a partir d'un fichièr .apport o .crash donat puslèu qu'a "
"partir de los que son en espèra dins %s. (Implicit se lo fichièr es donat "
"coma argument unic.)"

#: ../apport/ui.py:1117
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"En mòde de senhalament de bug, enregistra las informacions collectadas dins "
"un fichièr puslèu que de generar un rapòrt. Aqueste fichièr poirà èsser "
"utilizat pus tard a partir d'una autra maquina per mandar un rapòrt."

#: ../apport/ui.py:1133
msgid "Print the Apport version number."
msgstr "Aficha lo numèro de version d'Apport."

#: ../apport/ui.py:1292
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Aquò va aviar apport-retrace dins una fenèstra de terminal per examinar lo "
"crash."

#: ../apport/ui.py:1296
msgid "Run gdb session"
msgstr "Aviar una session gdb"

#: ../apport/ui.py:1297
msgid "Run gdb session without downloading debug symbols"
msgstr "Executar la session gdb sens telecargar los simbòls de desbugatge"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1299
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Mesa a jorn de %s amb una traça de la pila totalament simbolica"

#: ../apport/ui.py:1353
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"L’enregistrament de l’estat dels rapòrts de plantatge a fracassat. "
"Definicion impossibla del mòde de senhalament ;automatic o desactivat."

#: ../apport/ui.py:1357
msgid "Can't remember send report status settings"
msgstr ""
"Impossible de se remembrar dels paramètres d'estat del rapòrt de mandadís"

#: ../apport/ui.py:1439 ../apport/ui.py:1452
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Aqueste rapòrt d'incident s'aplica a un logicial qu'es pas pus installat per "
"aquesta distribucion."

#: ../apport/ui.py:1472
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Lo problèma concernís lo programa %s qu'a cambiat dempuèi lo plantatge."

#. can happen with broken core dumps
#. can happen with broken gz values
#: ../apport/ui.py:1531 ../apport/ui.py:1643 ../apport/ui.py:1843
msgid "This problem report is damaged and cannot be processed."
msgstr "Aqueste rapòrt d'incident es damatjat e pòt pas èsser tractat."

#: ../apport/ui.py:1537
msgid "This report is about a package that is not installed."
msgstr "Aqueste senhalament es a prepaus d’un paquet qu’es pas installat."

#: ../apport/ui.py:1544
msgid "An error occurred while attempting to process this problem report:"
msgstr ""
"Una error s'es producha al moment del tractament d'aqueste rapòrt "
"d'anomalia :"

#: ../apport/ui.py:1561
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""
"Avètz doas version installadas d’aquesta aplicacion, per la quala volètz "
"senhalar un bug ?"

#: ../apport/ui.py:1566
#, python-format
msgid "%s snap"
msgstr "snap %s"

#: ../apport/ui.py:1567
#, python-format
msgid "%s deb package"
msgstr "paquet deb %s"

#: ../apport/ui.py:1605
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""
"%s es provesit per un snap publicat per %s. Contactatz-los via %s per "
"obténer d’ajuda."

#: ../apport/ui.py:1610
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s es provesit per un snap publicat per %s. Cap d’adreça de contacte es pas "
"fornida ; consultatz lo forum a l’adreça https://forum.snapcraft.io/ per "
"obténer d’ajuda."

#: ../apport/ui.py:1696
msgid "Could not determine the package or source package name."
msgstr "Impossible de determinar lo nom del paquet o del paquet font."

#: ../apport/ui.py:1727
msgid "Unable to start web browser"
msgstr "Impossible d'aviar lo navigador web"

#: ../apport/ui.py:1728
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Impossible d'aviar lo navigador web per dobrir %s."

#: ../apport/ui.py:1801
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"Impossible de se connectar a la basa de donadas dels plantatges. Verificatz "
"vòstra connexion a l'Internet."

#: ../apport/ui.py:1804
msgid "Network problem"
msgstr "Problèma de ret"

#: ../apport/ui.py:1830
msgid "Memory exhaustion"
msgstr "Memòria saturada"

#: ../apport/ui.py:1832
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Vòstre sistèma dispausa pas de pro de memòria per tractar aqueste rapòrt "
"d'incident."

#: ../apport/ui.py:1877
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Lo problèma pòt pas èsser senhalat :\n"
"\n"
"%s"

#: ../apport/ui.py:1931 ../apport/ui.py:1943
msgid "Problem already known"
msgstr "Problèma ja conegut"

#: ../apport/ui.py:1933
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Aqueste problèma ja es estat senhalat dins lo rapòrt d'incident afichat per "
"vòstre navigador. Verificatz se podètz apondre d'informacions "
"complementàrias susceptiblas d'ajudar los desvolopaires."

#: ../apport/ui.py:1944
msgid "This problem was already reported to developers. Thank you!"
msgstr "Aqueste problèma es ja estat senhalat als desvolopaires. Mercé !"

#: ../bin/apport-cli.py:90
msgid "Press any key to continue..."
msgstr "Quichatz sus una tòca per contunhar..."

#: ../bin/apport-cli.py:97
msgid "What would you like to do? Your options are:"
msgstr "Qué volètz far ? Las causidas possiblas son :"

#: ../bin/apport-cli.py:110
#, python-format
msgid "Please choose (%s):"
msgstr "Fasètz una causida (%s) :"

#: ../bin/apport-cli.py:170
#, python-format
msgid "(%i bytes)"
msgstr "(%i octets)"

#: ../bin/apport-cli.py:172 ../gtk/apport-gtk.py:136 ../kde/apport-kde.py:417
msgid "(binary data)"
msgstr "(donadas binàrias)"

#: ../bin/apport-cli.py:212 ../gtk/apport-gtk.py:176 ../gtk/apport-gtk.ui.h:6
#: ../kde/apport-kde.py:208
msgid "Send problem report to the developers?"
msgstr "Mandar un rapòrt d'errors als desvolopaires ?"

#: ../bin/apport-cli.py:214
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Un còp que lo rapòrt d'incident serà estat mandat, completatz lo\n"
"formulari dobèrt automaticament dins vòstre navigador Web."

#: ../bin/apport-cli.py:221
#, python-format
msgid "&Send report (%s)"
msgstr "&Mandar lo rapòrt (%s)"

#: ../bin/apport-cli.py:225
msgid "&Examine locally"
msgstr "&Examinar en local"

#: ../bin/apport-cli.py:229
msgid "&View report"
msgstr "&Veire lo rapòrt"

#: ../bin/apport-cli.py:231
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&Conservar lo fichièr del rapòrt per lo mandar ulteriorament o lo copiar "
"endacòm"

#: ../bin/apport-cli.py:234
msgid "Cancel and &ignore future crashes of this program version"
msgstr ""
"Anullar e &ignorar los plantatges venents d'aquesta version del programa"

#: ../bin/apport-cli.py:237 ../bin/apport-cli.py:317 ../bin/apport-cli.py:353
#: ../bin/apport-cli.py:374
msgid "&Cancel"
msgstr "&Anullar"

#: ../bin/apport-cli.py:258
msgid "Problem report file:"
msgstr "Fichièr del rapòrt d'incident :"

#: ../bin/apport-cli.py:264 ../bin/apport-cli.py:269
msgid "&Confirm"
msgstr "&Confirmar"

#: ../bin/apport-cli.py:268
#, python-format
msgid "Error: %s"
msgstr "Error : %s"

#: ../bin/apport-cli.py:274 ../kde/apport-kde.py:465
msgid "Collecting problem information"
msgstr "Collècta d'informacions ligadas al problèma"

#: ../bin/apport-cli.py:276
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Las informacions collectadas pòdon èsser mandadas als desvolopaires per \n"
"melhorar l'aplicacion. Aquò pòt prene qualques minutas."

#: ../bin/apport-cli.py:292 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:516
msgid "Uploading problem information"
msgstr "Mandadís de las informacions ligadas al problèma"

#: ../bin/apport-cli.py:294
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Las informacions collectadas son mandadas al sistèma de seguit de bugs.\n"
"Aquò pòt prene qualques minutas."

#: ../bin/apport-cli.py:315
msgid "&Yes"
msgstr ""

#: ../bin/apport-cli.py:316
msgid "&No"
msgstr ""

#: ../bin/apport-cli.py:352
msgid "&Done"
msgstr "&Acabat"

#: ../bin/apport-cli.py:358
msgid "none"
msgstr "pas cap"

#: ../bin/apport-cli.py:359
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Seleccionat : % s. Causida multipla :"

#: ../bin/apport-cli.py:375
msgid "Choices:"
msgstr "Causida multipla :"

#: ../bin/apport-cli.py:389
msgid "Path to file (Enter to cancel):"
msgstr "Emplaçament del fichièr (tòca Entrada per anullar) :"

#: ../bin/apport-cli.py:395
msgid "File does not exist."
msgstr "Lo fichièr existís pas."

#: ../bin/apport-cli.py:397
msgid "This is a directory."
msgstr "Aquò's un repertòri."

#: ../bin/apport-cli.py:402
msgid "To continue, you must visit the following URL:"
msgstr "Per contunhar, vos cal anar a aquela adreça :"

#: ../bin/apport-cli.py:404
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Podètz dobrir un navigador Web ara, o picar aquesta URL dins un navigador "
"sus un autre ordenador."

#: ../bin/apport-cli.py:409
msgid "Launch a browser now"
msgstr "Aviar un navigador Web ara"

#: ../bin/apport-cli.py:424
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Pas cap de rapòrt d'incident en espèra. Ensajatz --help per mai "
"d'informacions."

#: ../bin/apport-retrace.py:48
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr ""
"Plaçar pas las « traças » novèlas dins lo rapòrt de bug, mas los escriure "
"sus la sortida estandarda."

#: ../bin/apport-retrace.py:55
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Avia una session gdb interactiva amb l'imatge memòria (core dump) del rapòrt "
"(-0 ignorat ; reescriu pas lo rapòrt)"

#: ../bin/apport-retrace.py:64
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Escriu lo rapòrt dins un fichièr donat puslèu que de modificar lo rapòrt "
"d'origina."

#: ../bin/apport-retrace.py:73
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Suprimir l'imatge memòria (core dump) del compte rendut aprèp regeneracion "
"de l'estat de la pila"

#: ../bin/apport-retrace.py:76
msgid "Override report's CoreFile"
msgstr "Espotir lo fichièr d'imatge memòria (CoreFile) del rapòrt"

#: ../bin/apport-retrace.py:79
msgid "Override report's ExecutablePath"
msgstr "Espotir l'emplaçament de l'executable (ExecutablePath) del rapòrt"

#: ../bin/apport-retrace.py:82
msgid "Override report's ProcMaps"
msgstr "Espotir la lista dels processus (ProcMaps) del rapòrt"

#: ../bin/apport-retrace.py:88
msgid "Rebuild report's Package information"
msgstr "Tornar construire las informacions suls paquets del rapòrt"

#: ../bin/apport-retrace.py:95
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Metre en plaça un nauc de sabla temporari e i telecargar/installar los "
"paquets e los simbòls de desbugatge necessaris ; sens aquesta opcion, serà "
"supausat que los paquets e los simbòls de desbugatge necessaris son ja "
"installats sul sistèma. L'argument punta cap al dorsièr de configuracion de "
"basa del sistèma de creacion de paquets ; s'especificatz \"system\",  los "
"fichièrs de configuracion del sistèma seràn utilizats, mas sols los "
"plantatges de la version en cors d'execucion poiràn èsser traçats."

#: ../bin/apport-retrace.py:109
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"Construire un autre nauc de sabla temporari per l'installacion de GDB e de "
"sas dependéncias en utilizant la meteissa version que la del rapòrt puslèu "
"que la version de GDB qu'avètz installada."

#: ../bin/apport-retrace.py:119
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Senhalar l'avançament del telecargament/de l'installacion pendent "
"l'installacion de paquets dins lo nauc de sabla."

#: ../bin/apport-retrace.py:125
msgid "Prepend timestamps to log messages, for batch operation"
msgstr ""
"Prefixar un orodatatge als messatges de jornal, per un tractament per lòts"

#: ../bin/apport-retrace.py:131
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""
"Crear e utilizar de depauses tèrces a partir d'originas especificadas dins "
"de rapòrts"

#: ../bin/apport-retrace.py:139
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Repertòri de cache pels paquets telecargats dins lo nauc de sabla"

#: ../bin/apport-retrace.py:145
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Repertòri pels paquets descompressats. Las execucions futuras presumiràn "
"qu'un paquet ja telecargat serà tanben extrait dins aqueste nauc de sabla."

#: ../bin/apport-retrace.py:156 ../bin/apport-valgrind.py:102
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Installar un paquet suplementari dins lo nauc de sabla (pòt èsser "
"especificat mantun còp)"

#: ../bin/apport-retrace.py:163
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Emplaçant d'un fichièr que conten las informacions d'autentificacion a la "
"basa de donadas dels plantatges. Aquò's utilizat quand òm indica un ID de "
"plantatge per cargar l'estat de la pila (solzment se -g, -o o -s son pas "
"especificats)"

#: ../bin/apport-retrace.py:173
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Afichar l'estat de la pila reconstituida e demandar una confirmacion abans "
"de lo mandar a la basa de donadas dels plantatges."

#: ../bin/apport-retrace.py:181
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Emplaçament de la còpia de la banca de donadas SQLite (per defaut : pas de "
"verificacion de la còpia)"

#: ../bin/apport-retrace.py:188
msgid "Do not add StacktraceSource to the report."
msgstr "Apondre pas StacktraceSource al rapòrt."

#: ../bin/apport-retrace.py:201
msgid "You cannot use -C without -S. Stopping."
msgstr "Podètz pas utilizar -C sens -S. Arrèst."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:231
msgid "OK to send these as attachments? [y/n]"
msgstr "D'acòrdi per mandar aquò coma pèças juntas ? [y/n]"

#: ../bin/apport-unpack.py:35
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr "%(prog)s <rapòrt> <repertòri cibla>"

#: ../bin/apport-unpack.py:36
msgid "Report file to unpack"
msgstr "Fichièr de rapòrt de descompressar"

#: ../bin/apport-unpack.py:37
msgid "directory to unpack report to"
msgstr "repertòri per descompressar lo rapòrt"

#: ../bin/apport-unpack.py:86
msgid "Destination directory exists and is not empty."
msgstr "Lo dorsièr de destinacion existís ja e es pas void."

#: ../bin/apport-valgrind.py:43
msgid "See man page for details."
msgstr "Vejatz la pagina de manual per mai de detalhs."

#: ../bin/apport-valgrind.py:52
msgid "specify the log file name produced by valgrind"
msgstr "especifica lo nom del fichièr jornal produit per valgrind"

#: ../bin/apport-valgrind.py:58
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"reütilizar un repertòri de nauc de sabla (« sandbox ») (SDIR) creat "
"precedentament o, s'existís pas, lo crear"

#: ../bin/apport-valgrind.py:66
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"crear pas o reütilizar pas un repertòri sandbox per de simbòls de desbugatge "
"addicionals mas s'apuejar solament suls simbòls de desbugatge instaltats."

#: ../bin/apport-valgrind.py:75
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"reütilizar un repertòri de cache (CDIR) precedentament creat o, s'existís "
"pas, lo crear"

#: ../bin/apport-valgrind.py:84
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"afichar la progression del telecargament e de l'installacion al moment de "
"l'installacion de paquets dins lo nauc de sabla (« sandbox »)"

#: ../bin/apport-valgrind.py:91
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""
"l'executable qu'executa jos l'aisina memcheck de valgrind per detectar de "
"pèrdas de memòria"

#: ../bin/apport-valgrind.py:133
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Error : %s es pas un executable. Arrèst."

#: ../data/apportcheckresume.py:75
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Aquò s'es produit precedentament al moment d'una mesa en velha e a empachat "
"lo sistèma de se reaviar corrèctament."

#: ../data/apportcheckresume.py:80
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Aquò s'es produit precedentament al moment d'una mesa en velha perlongada e "
"a empachat lo sistèma de se reaviar corrèctament."

#: ../data/apportcheckresume.py:88
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Lo processus de represa s'es blocat juste abans d'arribar a son tèrme e "
"apareisserà coma s'èra acabat normalament."

#: ../data/kernel_oops.py:34
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Ara, vòstre sistèma se pòt far instable e pòt aver besonh d'èsser reaviat."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Senhalar un problèma..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Senhalar un disfoncionament als desvolopaires"

#: ../gtk/apport-gtk.py:153
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "O planhèm, %s s'es arrestada d'un biais inesperat."

#: ../gtk/apport-gtk.py:156
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "O planhèm, %s a quitat d'un biais imprevist."

#: ../gtk/apport-gtk.py:161 ../kde/apport-kde.py:223 ../kde/apport-kde.py:272
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "O planhèm, %s a rencontrat una error intèrna."

#: ../gtk/apport-gtk.py:184 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:216
msgid "Send"
msgstr "Mandar"

#: ../gtk/apport-gtk.py:206 ../gtk/apport-gtk.py:620 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:333
msgid "Show Details"
msgstr "Afichar los detalhs"

#: ../gtk/apport-gtk.py:219 ../kde/apport-kde.py:264 ../kde/apport-kde.py:279
msgid "Continue"
msgstr "Contunhar"

#: ../gtk/apport-gtk.py:243
#, python-format
msgid "The application %s has stopped responding."
msgstr "L'aplicacion %s a daissat de respondre."

#: ../gtk/apport-gtk.py:247
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Lo programa \"%s\" a daissat de respondre."

#: ../gtk/apport-gtk.py:264 ../kde/apport-kde.py:231
#, python-format
msgid "Package: %s"
msgstr "Paquet : %s"

#: ../gtk/apport-gtk.py:270 ../kde/apport-kde.py:238
msgid "Sorry, a problem occurred while installing software."
msgstr ""
"O planhèm, una error s'es produita al moment de l'installacion del logicial."

#: ../gtk/apport-gtk.py:280 ../gtk/apport-gtk.py:299 ../kde/apport-kde.py:246
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "L'aplicacion %s a patit una error intèrna."

#: ../gtk/apport-gtk.py:282 ../kde/apport-kde.py:251
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "L'aplicacion %s a quitat d'un biais imprevist."

#: ../gtk/apport-gtk.py:307 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:276
msgid "If you notice further problems, try restarting the computer."
msgstr "En cas de problèmas mai, ensajatz de reaviar vòstre ordenador."

#: ../gtk/apport-gtk.py:310 ../kde/apport-kde.py:281
msgid "Ignore future problems of this type"
msgstr "Ignorar los problèmas d'aquesta mena que venon"

#: ../gtk/apport-gtk.py:624 ../kde/apport-kde.py:330
msgid "Hide Details"
msgstr "Amagar los detalhs"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:384 ../kde/apport-kde.py:558
msgid "Apport"
msgstr "Apòrt"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Abandonar"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "D'acòrdi"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Rapòrt d'anomalia"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>O planhèm, una error intèrna s'es produita !</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Se'n remembrar d'ara enlà"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorar los problèmas que venon d'aquesta version del programa"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Reaviar aquesta aplicacion"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Examinar en local"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "Mandar pas"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Collècta d'informacions ligadas al problèma</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"De donadas son en cors de collècta per ajudar los desvolopaires a corregir "
"lo problèma qu'avètz rencontrat."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Mandadís de las informacions ligadas a l'incident</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:518
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Las donadas collectadas son transmesas al sistèma de seguit dels bugs. Aquò "
"pòt prene qualques minutas."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "Fichièr d'arrèst brutal d'Apport"

#: ../kde/apport-kde.py:267
msgid "Leave Closed"
msgstr "Daissar tampat"

#: ../kde/apport-kde.py:268 ../kde/apport-kde.py:435
msgid "Relaunch"
msgstr "Reaviar"

#: ../kde/apport-kde.py:361
msgid "Username:"
msgstr "Nom d'utilizaire :"

#: ../kde/apport-kde.py:364
msgid "Password:"
msgstr "Senhal :"

#: ../kde/apport-kde.py:464
msgid "Collecting Problem Information"
msgstr "Collècta de las informacions ligadas al problèma"

#: ../kde/apport-kde.py:467
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Las informations collectadas pòdon èsser mandadas als desvolopaires per "
"melhorar l'aplicacion. Aquò pòt prene qualques minutas."

#: ../kde/apport-kde.py:515
msgid "Uploading Problem Information"
msgstr "Mandadís de las informacions ligadas al problèma"
